code/ATMOSPHERICS/components/unary_devices/vent_pump.dm

```diff
@@ -263,7 +263,7 @@
 		return 1
 	if(istype(W, /obj/item/weapon/weldingtool))
 		var/obj/item/weapon/weldingtool/WT = W
-		if (WT.remove_fuel(0,user))
+		if (WT.remove_fuel(1,user))
```

Comment:
Pretty sure this means that if the user is interrupted mid-welding, they'll have to spend one more fuel unit to try a second time.